zips/zip-0228.rst

```diff
@@ -2,12 +2,539 @@
+|                             |                          |                                           |:math:`\mathsf{rcv}` values of commitments in the Order.             |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``varies``                   |``SwapActionGroup``       |``ActionGroupDescription``                 |The description of the Action Group corresponding to the desired     |  
+|                             |                          |                                           |Swap operation. It must contain at least two Actions, the Swap       |  
+|                             |                          |                                           |Action and the Fee Action.                                           |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+
+The ``SwapIO`` data type is used to represent the input and output of a Swap Order.
+
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+| Bytes                       | Name                     | Data Type                                 | Description                                                         |
++=============================+==========================+===========================================+=====================================================================+
+|``32``                       |``asset_base``            |``byte[32]``                               |The Asset Base [#zip-0227-assetbase]_ of the input/output of the     |  
+|                             |                          |                                           |Swap Order.                                                          |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``8``                        |``qty``                   |``uint64``                                 |The quantity of the Asset represented by ``asset_base`` proposed in  |  
+|                             |                          |                                           |the Swap Order.                                                      |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+
+Rationale for Swap Orders
+-------------------------
+
+In a Swap Order, senders only manipulate their funds. 
```

Comment:
`In a Swap Order, users revel information about the content of their offer` not sure about `only manipulate their funds.

Reply:
Edited.